admin/src/components/Button/Index.tsx

```diff
@@ -0,0 +1,29 @@
+import clsx from "clsx";
+import { ComponentPropsWithoutRef } from "react";
+import Spinner from "../Spinner";
+
+interface ButtonProps extends ComponentPropsWithoutRef<"button"> {
+  loading?: boolean;
+  styledClassName?: string;
+}
+
+export const Button = ({
+  children,
+  loading,
+  disabled,
+  styledClassName,
+  ...rest
+}: ButtonProps) => {
+  return (
+    <button
+      {...rest}
```

Comment:
in this case you would have to have rest after classname and your component wouldn't be a button that can be replaced, it would always have the default classes unless you use !important